src/DataGrid.tsx

```diff
@@ -94,13 +94,13 @@ export interface DataGridProps<R, K extends keyof R> {
   rowGroupRenderer?: React.ComponentType;
   emptyRowsView?: React.ComponentType<{}>;
   /** Component used to render a draggable header cell */
-  draggableHeaderCell?: React.ComponentType<{ column: CalculatedColumn<R>; onHeaderDrop(): void }>;
+  draggableHeaderCell?: React.ComponentType<{ column: CalculatedColumn<R, never>; onHeaderDrop(): void }>;
```

Comment:
```suggestion
  draggableHeaderCell?: React.ComponentType<{ column: CalculatedColumn<R, SR>; onHeaderDrop(): void }>;
```
Why use `never` here?